apps/webapp/package.json

```diff
@@ -32,6 +32,7 @@
     "zustand": "^4.5.1"
   },
   "devDependencies": {
+    "@penumbra-zone/getters": "workspace:*",
```

Comment:
This should be a normal dependency given it's a runtime package and not really dev tooling